docs/src/main/asciidoc/rest-json.adoc

```diff
@@ -186,6 +186,11 @@ It will allow to narrow down the number of JAX-RS providers (which can be seen a
 
 ==== Jackson
 
+In Quarkus, Jackson is configured to ignore the unknown properties (by disabling the `DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES` feature).
```

Comment:
You might want to add a note that this only works for the ObjectMapper configured by Quarkus and which they obtain via CDI (or use implicitly via RESTEasy etc).

I know it should be evident, but I am sure it might come as a surprise to some users.